java/8/Dockerfile

```diff
@@ -26,9 +26,11 @@
 # issue is backported or if Forge decides to abandon old versions and we need
 # newer Java versions. Old Quay images were updated like once a year, so it's ok.
 #
+# Update: This issue has been resolved, see issue below for more info - No767 (Noelle)
```

Comment:
The issue is not resolved - if it is, all the comments could be removed. There's no backport as far as I am aware?

Removing this will break everyone's servers if they're not on >= `1.16.5 - 36.2.26`

Reply:
i think both the java 8 and 11 images can be kept on those exact images (the old and current openjdk base images). i'll update it to keep it on those versions

Reply:
You can replace them with the temurin alternatives, e.g I use `eclipse-temurin:8u312-b07-jdk-focal`

Reply:
What modpacks have this issue? I've absolutely launched 1.7.10 and 1.12.2 servers using the latest temurin java 8 release.